docker/install/ubuntu_install_ethosu_driver_stack.sh

```diff
@@ -78,6 +78,17 @@ mkdir -p "${ethosu_dir}"
 cd "${ethosu_dir}"
 git clone --branch ${ethosu_driver_ver} "https://review.mlplatform.org/ml/ethos-u/ethos-u-core-driver" core_driver
 git clone --branch ${ethosu_driver_ver} "https://review.mlplatform.org/ml/ethos-u/ethos-u-core-platform" core_platform
+# Prepare docker image for driver updates
+git clone --branch 22.08 "https://review.mlplatform.org/ml/ethos-u/ethos-u-core-driver" core_driver_22_08
+git clone --branch 22.08 "https://review.mlplatform.org/ml/ethos-u/ethos-u-core-platform" core_platform_22_08
```

Comment:
Curious, why do we need to build both the 21.11 and 22.08 version?

Reply:
Version 21.11 is needed to pass tests because there were changes in version 22.08 that could not be checked without updating docker image. As I understand it, after accepting this PR, it will be possible to check PR https://github.com/apache/tvm/pull/13529.

Reply:
Thanks, so if I understood correctly we can remove installation of the old version in https://github.com/apache/tvm/pull/13529?

Reply:
Yes, that's right.